uvm/impl/com/untangle/uvm/AppTCPSessionImpl.java

```diff
@@ -37,6 +41,23 @@ public class AppTCPSessionImpl extends AppSessionImpl implements AppTCPSession
     protected boolean[] lineBuffering = new boolean[] { false, false };
     protected ByteBuffer[] readBuf = new ByteBuffer[] { null, null };
 
+    /**
+     * Size of buffer to work with.
+     * The faster the network and ability to process, the bigger the size performs throughput impact
+     */
+    private static int Buffer_size = 8 * 1024;
```

Comment:
This same value is used in TCPSource.java. Could it be global?

Reply:
Not easily, but the point of specifying the tcp_buffer_size value from the command line essentially makes it global.

Reply:
Makes sense to me! Resolving